pulp_ansible/app/models.py

```diff
@@ -24,7 +25,7 @@
 log = getLogger(__name__)
 
 
-class Role(Content):
+class Role(Content, AutoAddObjPermsMixin):
```

Comment:
You should remove the `AutoAddObjPermsMixin` from here and CollectionVersion model.